Tests/GeoLib/TestPolygon.cpp

```diff
@@ -199,6 +199,28 @@ TEST_F(PolygonTest, isPolylineInPolygon)
 		delete pnts[k];
 }
 
+TEST_F(PolygonTest, AssignmentOperator)
+{
+	// create a polygon
+	GeoLib::Polyline polyline(_pnts);
+	polyline.addPoint(0);
+	polyline.addPoint(1);
+	polyline.addPoint(2);
+	polyline.addPoint(0);
+	GeoLib::Polygon polygon(polyline);
+	// assign an other polygon
+	polygon = *_polygon;
+	// ToDo change to segments
```

Comment:
Very minor thing: I suggest to agree on one spelling for TODO. E.g., my IDE recognizes `TODO` and gathers all those lines in a nice list.